src/api/app/views/webui/comment/_comment_list.html.haml

```diff
@@ -1,3 +1,5 @@
+- commentable.bs_request_actions.flat_map { |a| a.comments.without_parent.includes(:user) }.each do |comment|
```

Comment:
I believe this line is doing several queries, right? One for the `commentable.bs_request_actions.flat_map { ... }` and another for each of the `a.comments.without_parent.includes(:user)`. Can we rewrite it? I think querying for `Comments.where(...)` will fix that.

Reply:
I "borrowed" that line from here: https://github.com/openSUSE/open-build-service/blob/e0ffa8e1d498e43a3e5806f30592e97be8fdf380/src/api/app/components/bs_request_activity_timeline_component.rb#L13

And I used it again here: https://github.com/openSUSE/open-build-service/blob/e0ffa8e1d498e43a3e5806f30592e97be8fdf380/src/api/app/controllers/comments_controller.rb#L6

Tests are not passing. After I fix them, I will take a look if it can be improved.

Reply:
I tried with `Comment.without_parent.where(commentable: BsRequestAction.where(bs_request: commentable))`. It seems to work. Please take a look again.

Reply:
Nice